src/styles/card.scss

```diff
@@ -0,0 +1,113 @@
+@import "./variables.scss";
```

Comment:
By convention, partial files (containing reusable code) in SCSS are given names that begin with an underscore (e.g. _variables.scss) to indicate that they should not be compiled into a standalone CSS file.